test/functionalTest/cases/725_multiservice_ignore/service_ignore_without_multiservice.test

```diff
@@ -0,0 +1,103 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Ignoring service header when not running in multiservice mode
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255 IPV4
+
+--SHELL--
+echo "1. Create an entity using foo as tenant"
+echo "======================================="
+url="/v1/updateContext"
+payload='<?xml version="1.0" encoding="UTF-8"?>
+<updateContextRequest>
+  <contextElementList>
+    <contextElement>
+      <entityId type="Test" isPattern="false">
+        <id>E</id>
+      </entityId>
+      <contextAttributeList>
+        <contextAttribute>
+          <name>att</name>
+          <type>compound</type>
+          <contextValue>03</contextValue>
+        </contextAttribute>
+      </contextAttributeList>
+    </contextElement>
+  </contextElementList>
+  <updateAction>APPEND</updateAction>
+</updateContextRequest>'
+orionCurl --url "$url" --payload "$payload" --httpTenant "foo"
+echo
+echo
+
+echo "2. Check that the number of orion databases is just 1"
+echo "====================================================="
+# In this case mongoCmd helper is not used, due to we don't have
+# a particular database for mongo command and the output could be
+# multiline in a general case
+echo "show dbs" | mongo | grep ${CB_DATABASE_NAME} | wc -l
+
```

Comment:
So, I imagine that here you want to prove that "orion-foo" hasn't been created.

Is the DB 'orion' created even before any content has been sent?
I mean, there is a possibility that *only orion-foo* exists here. It depends on the implementation ...